packages/reference/src/resources/SingleResourceReferenceEditor.tsx

```diff
@@ -17,10 +18,18 @@ export function SingleResourceReferenceEditor(
   }
 ) {
   const { dialogs, field } = props.sdk;
+
+  const editorPermissions = useEditorPermissions({
```

Comment:
we could move `useEditorPermissions` inside of `useResourceLinkActions` as it's the same for single nad multiple resources :)

+ NIT: instead of passing all props down, just provide the necessary ones :)

Reply:
@chrishelgert I wasn't sure where else this is used, changed it 👍